### PR TITLE
new domain request

### PR DESCRIPTION
**Pull request by kingraphaii**
### Link to Website

Links
- Current
  - https://gatewaystream.com
  - gsts-dev-01.is-an.app (requested for new app backend) 

---

- [x] Toggle this checkbox to re-run the stargazer checking

### CODE REVIEW

File: domains/gsts-dev-01.is-an.app.json [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+    "domain": "is-an.app",
+    "subdomain": "gsts-dev-01",
+    "owner": {
+      "repo": "https://github.com/gateway-stream/gateway_be",
+      "email": "percival.rapha@gmail.com"
+    },
+    "record": {
+      "A": ["34.201.17.130"]
+    },
+    "proxy": false
```
Comment (tarampampam) [inline domains/gsts-dev-01.is-an.app.json:13]: Could you let me know if you are sure about that? Proxying can help protect your backend

**Comment by tarampampam** [comment]
And one more question - since you already have a domain `gatewaystream.com` - why didn't you make a subdomain like `gsts-dev-01.gatewaystream.com`?

**Comment by tarampampam** [comment]
ping @kingraphaii

File: domains/gsts-dev-01.is-an.app.json (hunk 1 above)
Comment (kingraphaii) [reply domains/gsts-dev-01.is-an.app.json:13]: What kind of limitations, is this the same as Cloudflare Proxying?

Comment (tarampampam) [reply domains/gsts-dev-01.is-an.app.json:13]: This is it

**Comment by kingraphaii** [comment]
If it's like cloudflare proxying, then leave it as is.


> And one more question - since you already have a domain `gatewaystream.com` - why didn't you make a subdomain like `gsts-dev-01.gatewaystream.com`?

I do not have access to the dns managemet console, so i'm stuck with using another service for now.

**Pull request merged**